grpc/types.go

```diff
@@ -97,6 +99,28 @@ func (m Method) ResponseName() string {
 
 var EmptyReq = Message{
 	Name: "EmptyReq",
+	Args: []Argument{
```

Comment:
I think we should give it a new name, try not to use 'empty'.

Reply:
OK. Rename to TransactionReq